### PR TITLE
[memorycache] Support unaligned memory access

### PR DESCRIPTION
**Pull request by cbiesinger**
Let me know if this indentation of the second line of the if is OK

### CODE REVIEW

File: extensions/memorycache/memorycache.js [hunk 1]
```diff
@@ -149,7 +153,8 @@ Loader.OnLoad(function() {
         address = new PointerMath.Pointer(address).value();
 
         var viewer = getArrayViewer(size, isUnsigned, isFloat);
-        if (!loadPage(getPage(address), viewer, function(view) {
+        if (isUnaligned(address, size) ||
```
Comment (sanketj) [inline extensions/memorycache/memorycache.js]: Can we add a comment or two about this check and why it is needed? Both here and below.

Comment (cbiesinger) [reply extensions/memorycache/memorycache.js]: Done

Comment (sanketj) [reply extensions/memorycache/memorycache.js]: LGTM. Please get sign off from @petersalas as well.

**Comment by cbiesinger** [comment]
I'm sorry, I saw your comment too late :(

@petersalas , does this look OK to you? If not I can revert/fix

**Comment by petersalas** [comment]
LGTM!

**Pull request merged**